examples/passport-login/src/authentication-interceptors/facebook-interceptor.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
```

Comment:
Nitpick: we typically name the file as `facebook.interceptor.ts`.

Reply:
agreed